docs-ref-conceptual/upcoming-breaking-changes.md

```diff
@@ -63,6 +63,12 @@ The [db-up](https://github.com/Azure/azure-cli-extensions/tree/main/src/db-up) e
 
 - This command group has been deprecated and will be removed in next breaking change release(2.67.0). Use 'az network public-ip ddos-protection' instead.
 
+## profile
+
+### `az login`
+
+- Passing the service principal certificate with `--password` is deprecated and will be removed in version 2.67.0. Please use `--certificate` instead.
```

Comment:
```suggestion
- The `--password` parameter will not accept service principal certificate starting with version 2.67.0, use the `--certificate` instead.
```

Reply:
Is the `the` before `--certificate` needed?

Reply:
Maybe we can align with https://github.com/Azure/azure-cli/issues/30283?